besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1946,15 +1962,16 @@ private Path pluginsDir() {
     }
   }
 
-  public File nodePrivateKeyFile() {
-    File nodePrivateKeyFile = null;
-    if (isFullInstantiation()) {
-      nodePrivateKeyFile = standaloneCommands.nodePrivateKeyFile;
-    }
+  private NodeKey buildNodeKey() {
+    addConfigurationService(); // its a no-op if besu configuration service is already initialized
```

Comment:
this is a side effect and not the purpose of this function - I _think_ it'd be better for this to be called explicitly when needed.

Reply:
how do you recommend we call addConfigurationService from subcommand? I would either have to expose addConfigurationService as public or wrap these two methods in a different method.

Reply:
will use another method reference!